operator/v1/types_ingress.go

```diff
@@ -319,6 +319,11 @@ type NodePlacement struct {
 	//
 	// These defaults are subject to change.
 	//
+	// Note that using nodeSelector.matchExpressions is not supported.  Only
+	// nodeSelector.matchLabels may be used.  This is a limitation of the
+	// Kubernetes API: the pod spec does not allow complex expressions for
```

Comment:
Is there an upstream issue we could reference either here or in the PR description?